resources/views/device-group/index.blade.php

```diff
@@ -55,6 +55,35 @@ class="fa fa-trash" aria-hidden="true"></i></button>
                     </table>
                 </div>
             </div>
+            <div class="panel-heading">
```

Comment:
```suggestion
        </div>
    </div>
    <div class="container-fluid">
        <div id="manage-device-groups-panel" class="panel panel-default">
            <div class="panel-heading">
```

Reply:
Maybe something like that to separate them out slightly:

![image](https://user-images.githubusercontent.com/3941142/63551827-0d129580-c52e-11e9-938f-e19261c6e044.png)

Reply:
@laf 
i agree with your idea.
Could you commit this to my pull request please?

Reply:
Says I don't have permission. You have to enable that in the pull request (can't remember the option), otherwise I think you can accept my suggestion above.

Reply:
strange, "allows edit from maintainers" is checked